listener.c

```diff
@@ -433,10 +433,8 @@ listener_read_cb(evutil_socket_t fd, short what, void *p)
 		++lev->refcnt;
 		cb = lev->cb;
 		user_data = lev->user_data;
-		UNLOCK(lev);
```

Comment:
And what about `errrocb`?

https://github.com/libevent/libevent/blob/89505f8513b8590e7a7700d481c20562d0726d54/listener.c#L461-L463

Reply:
U're absolutely right!  Errorcb was not in my project, but I'll check it .